mysql-plugin/widgets/Mysql-batchsink.json

```diff
@@ -0,0 +1,127 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "icon": {
```

Comment:
icon should be built by the maven plugin (through files in the icons directory) and not specified inline here.

Reply:
See https://github.com/data-integrations/google-cloud as an example